src/lib/pipeline/m2/index.js

```diff
@@ -20,23 +21,31 @@ class M2 extends THREE.Group {
 
     const bones = [];
     const rootBones = [];
+    const indexedBones = [];
 
-    this.data.bones.forEach((joint) => {
+    this.data.bones.forEach((joint, index) => {
       const bone = new THREE.Bone();
 
-      const { pivotPoint } = joint;
+      indexedBones[index] = bone;
```

Comment:
Gotcha. I think I have the same ordering in indexedBones as would have existed in bones in the first place. Woops!

I'll remove that.